### ISSUE TITLE
I'm running python3 and command prompt python server3.py is not giving any response.

### ISSUE THREAD

**Issue by KamalSagar** [insidesherpa/JPMC-tech-task-1-PY3#132]
I am running server3.py command on cmd prompt and the server3.py file was open and there is a error in the code of this file at lines 223 and 224.

**Comment by rahulmansharamani14** [comment]
The same goes for me too. No output is shown after running both commands on separate cmd prompt.

**Comment by KamalSagar** [comment]
do the whole process from download task 1 and then run it works. firstly run server3.py on admin cmd prompt and after that run client on another nonadmin cmd prompt

**Comment by francisco50** [comment]
The solution is once you get to that point on running server3.py and client3.py, open two different command prompts, at least one of them you should run as administrator and the other should be normal. once the other one starts the other will start too. The idea is the server starts and the client gets the connection from the main source.

**Comment by rahulmansharamani14** [comment]
After they start both after a long while they didn't stop and I have to shut down command prompt forcibly. How should I know that I have completed the setup successfully?

**Comment by francisco50** [comment]
Yeah that means they are working fine. You just have to follow the next step. You can stop them to create your patch file after that.

**Comment by rahulmansharamani14** [comment]
I got it. Thanks, @francisco50 for guiding me through it.
